mono/mini/simd-intrinsics-netcore.c

```diff
@@ -721,14 +718,27 @@ emit_x86_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature
 			EMIT_NEW_ICONST (cfg, ins, supported ? 1 : 0);
 			ins->type = STACK_I4;
 			return ins;
-		//case SN_MultiplyNoFlags:
-			//// TODO: implement using _mulx_u32/u64:
-			//// ulong MultiplyNoFlags(ulong left, ulong right)
-			//// ulong MultiplyNoFlags(ulong left, ulong right, ulong* low) => MultiplyNoFlags(left, right, low);
-			//// uint MultiplyNoFlags(uint left, uint right)
-			//// uint MultiplyNoFlags(uint left, uint right, uint* low)
-			//return NULL;
-		//case SN_ZeroHighBits:
+		case SN_MultiplyNoFlags:
+			if (fsig->param_count == 2) {
+				MONO_INST_NEW (cfg, ins, is_64bit ? OP_MULX_H64 : OP_MULX_H32);
+				ins->dreg = alloc_ireg (cfg);
+				ins->sreg1 = args [0]->dreg;
```

Comment:
Needs to be alloc_lreg () for 64 bit.

Reply:
oops, thanks!

Reply:
I revised all methods (currently these intrinsics are emitted only for #if AMD64 so it was ok but I guess we can also enable them for x86 (once we have a test lane for it).